### DISCUSSION TITLE
How to embed content in Typemill? Iframes or other embedding options?

### DISCUSSION THREAD

**Discussion by lbarriocanal** [typemill/typemill#441] [Q&A] [answered]
Hi,
I´m trying to embed Google Slides or H5p content in Typemill pages but I get this alert:
_"Body not secure. For code please use markdown `inline-code`"_
I have read shortcodes tutorial but my technical knowledge is not good enough.
Is there some way to insert iframes in markdown?  Other embedding options?
Thanks everybody.

**Comment by trendschau** [comment, marked as answer]
Sorry for late reply. Yes, from security reasons direct input of html or iframes is not possible in Typemill. The proper way is a shortcode plugin. 

I am aware that these restrictions are very limiting for some users, so I will add a general iframe integration plugin to the list, where you can add iframes from relevant sources. I think it is a good plugin for the new license that will be introduced in some weeks.

**Reply by lbarriocanal** [reply]
Ok. Thanks a lot.
